pkgs/development/compilers/flasm/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchurl, unzip, bison, flex, gperf, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "flasm";
+  version = "1.64";
+
+  src = fetchurl {
+    url = "https://www.nowrap.de/download/flasm16src.zip";
+    sha256 = "0n7sqv0b4f3wjfrhlv9d1ynza1chcmnrqxvpp0b3dh68ridnjdxv";
+  };
+
+  unpackPhase = ''
+    unzip $src -d src
+    cd src
+  '';
+
+  # Make build reproducible
```

Comment:
nix already garantee that `__DATE__` will be `Jan  1 1980`, so this patch is not needed.

Reply:
When I build it locally it seems to use the current date though, is it because I'm not sandboxing?

Reply:
I guess you disabled sandboxing, since `nix.useSandbox` is true by default.